templates/HAPROXY_HEAD

```diff
@@ -0,0 +1,18 @@
+global
+  daemon
+  log /dev/log local0
+  log /dev/log local1 notice
+  maxconn 4096
+  tune.ssl.default-dh-param 2048
+defaults
+  log               global
+  retries           3
+  maxconn           2000
+  timeout connect   5s
+  timeout client    50s
+  timeout server    50s
+listen stats
+  bind 0.0.0.0:9090
```

Comment:
That's reasonable but I don't understand how the templates get into the sandbox in the first place. Can you point me to the feature we would use for that?